CMakeLists.txt

```diff
@@ -52,14 +52,14 @@ find_package(clang)
 
 if(NOT MSVC)
     find_library(LIBXML2 NAMES xml2 libxml2)
-    if(${LIBXML2} STREQUAL "LIBXML2-NOTFOUND")
```

Comment:
One of the worst parts of the CMake language is that all `*-NOTFOUND` strings evaluate to false.